### ISSUE TITLE
Missing « ?raw=true » in the PUT url when editing styles

### ISSUE THREAD

**Issue by MaelREBOUX** [georchestra/mapstore2-georchestra#254]
Working with a GeoServer 2.12.5.

I want to edit a layer style from MapStore2. The initial POST an PUT request (to copy / paste the current style into a temporary style on the GeoServer) work well : the SLD code arrives on the GeoServer.

But when changing the SLD code in the editor, a new PUT is triggered after 2 seconds. This PUT bugs : the style on the GeoServer is broken.

```xml
<?xml version="1.0" encoding="UTF-8"?>
<sld:StyledLayerDescriptor xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:se="http://www.opengis.net/se" xmlns:gml="http://www.opengis.net/gml" xmlns:ogc="http://www.opengis.net/ogc" xmlns:sld="http://www.opengis.net/sld">
org.geotools.styling.StyledLayerDescriptorImpl@763b1c69
</sld:StyledLayerDescriptor>
```

The single difference between the 2 PUT is the « ?raw=true » at the end of the url for the fisrt one.
When playing / testing the the RESTED plugin in FireFox and testing the same url with and without « ?raw=true » I can reproduced the bug.

So, in conclusion I think MapStore2 should add the « ?raw=true » string at the end systematically.
